src/script/script_gui.cpp

```diff
@@ -939,7 +939,7 @@ struct ScriptDebugWindow : public Window {
 
 	/**
 	 * Change all settings to select another Script.
-	 * @param show_ai The new AI to show.
+	 * @param show_script The new AI to show.
```

Comment:
This function changes the current script log into a log of another script, and if a script settings window is open, it closes it to avoid confusion. It can be AI or GS.